cli/cmd/completion.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/cmd/helm/require"
+)
+
+const completionDesc = `
+Generate autocompletion script for lokoctl for the specified shell (bash or zsh).
+`
+
+const bashCompDesc = "\nGenerate the autocompletion script for lokoctl for the bash shell. \n\n" +
+
+	"To load completions in your current shell session: \n" +
+	"`$ source <(lokoctl completion bash)` \n\n" +
+
+	"To load completions for every new session, execute once: \n" +
+	"Linux: `$ lokoctl completion bash > /etc/bash_completion.d/lokoctl && exec $SHELL` \n" +
+	"MacOS: `$ lokoctl completion bash > /usr/local/etc/bash_completion.d/lokoctl && exec $SHELL` "
+
+const zshCompDesc = "\nGenerate the autocompletion script for lokoctl for the zsh shell. \n\n" +
+
+	"To load completions in your current shell session: \n" +
+	"`$ source <(lokoctl completion zsh)` \n\n" +
+
+	"To load completions for every new session, execute once: \n" +
+	"`$ lokoctl completion zsh > \"${fpath[1]}/_lokoctl\" && exec $SHELL`"
+
+func newCompletionCmd(out io.Writer) *cobra.Command {
```

Comment:
Is this possible to mark parameter as not used?
```suggestion
func newCompletionCmd(_ io.Writer) *cobra.Command {
```

Reply:
But we need this writer. It is passed onto `runCompletionZsh` and `runCompletionBash`

Reply:
Ah, sorry, I missed that :+1: